README.md

```diff
@@ -1,25 +1,84 @@
-# PySATL-IGraph_CPD-Module
+# PySATL-CPD-Module
 
-Dependencies installation
+PLACE FOR BADGES
```

Comment:
About badges: https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/monitoring-workflows/adding-a-workflow-status-badge

Also you can see them in the neighboring projects:
1) https://github.com/toxakaz/EM-algo/blob/main/README.md?plain=1
2) https://github.com/SE-Processor-Fuzzing/control-hazard-analyzer/blob/main/README.md?plain=1